about.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet" />
+    <title>CREATIVE CULTURE SUMMIT</title>
+</head>
+
+<body>
+    <header>
+        <nav class="upper-nav">
+            <ul class="upper-ul">
+                <li><a href="#"><img src="./images/facebook.png" alt="facebookIcon"></a></li>
+                <li><a href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-navbar">
+            <span class="menu" id="menu"><i class="fas fa-bars"></i></span>
+            <a href="index.html"><img class="logo" src="./images/logo.png" alt="logo"></a>
+            <ul class="second-list" id="mobile-menu">
+                <li><i class="fas fa-times close-btn" id="close-btn"></i></li>
+                <li><a href="index.html" class="nav-list" id="about-link">Home</a></li>
+                <li><a href="#" class="nav-list">Program</a></li>
+                <li><a href="#" class="nav-list">Join</a></li>
+                <li><a href="#" class="nav-list">Sponsor</a></li>
+                <li><a href="#" class="nav-list">News</a></li>
+                <li><a href="#" class="button nav-list">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="main-container about-container">
+        <div class="center-about">
+            <h1 class="top-h1-about">Hello! sharing world</h1>
+            <h2 class="main-sec-heading-about">CREATIVE CULTURE GLOBAL SUMMIT 2015</h1>
+            <p class="main-par-about">
+                Ultimately, in the long run, the aim or objective Creative Culture Global Summit 2015 is 
+                to establish a shared opinion and vision about the road forward. 
+                Finding imperatives that could change. 
+                Helping to establish guidelines for future decision-making.
+            </p>
+            <p class="contact-us">
+                Please send us an email with your questions about CC Global Summit 2015! </p>
+            <a href="#" class="email-id">ccsummit2015@cccghana.org </a>
```

Comment:
- [ ] please, if you check well, you will see the upper section of the about page has a different background color from the lower part. Please add that.
- [ ] ensure the small text if a box has a white background 
- [ ] please personalize the surrounded text. Remember, you can personalize the text, but design should be followed.
- [ ] please make the main about page topic bolder.
your design
![Screenshot (1096)_LI](https://github.com/erickma1/CapstoneProject/assets/65298048/7e06195b-4732-434b-9850-40a4df8647bd)
original
![Screenshot (1097)_LI](https://github.com/erickma1/CapstoneProject/assets/65298048/978bbba8-3007-44f0-afa7-e9671fb856f1)